scss/book-list-detail.scss

```
@@ -0,0 +1,97 @@
+.body{
+  font-family: 'Roboto', sans-serif;
+}
+
+.master-container{
+  display: flex;
+  height: 768px;
+  width: 1366px;
+  background-color: $wild-sand;
+  justify-content: center;
+  align-items: center;
+
+  .navbar-container{
+      position: fixed;
+      top: 0;
+
+      .border{
+        height: 6px;
+        width: 1366px;
+        background-color: $cerulean;
+      }
+
+      .navbar{
+          height: 70px;
+          width: 1366px;
+          background-color: $white;
+          box-shadow: 0 4px 10px 0 rgba(0,0,0,0.5);
+
+          .wolox-logo{
+            position: absolute;
+            left: 246px;
+            right: 908px;
+            height: 70px;
+            width: 212.42px;
```

Remember to round up the decimals
```suggestion
            width: 212px;
```